pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -271,6 +273,15 @@ func (a *ApplicationBackupController) handle(ctx context.Context, backup *stork_
 			if !canDelete {
 				return nil
 			}
+			// get-rid of map entry for termination channel and rule flag
+			if _, ok := a.terminationChannels[string(backup.UID)]; ok {
+				log.ApplicationBackupLog(backup).Infof("deleted termination channel entry from controller map for backup [%v/%v]", backup.Namespace, backup.Name)
+				delete(a.terminationChannels, string(backup.UID))
+			}
+			if _, ok := a.execRulesCompleted[string(backup.UID)]; ok {
+				log.ApplicationBackupLog(backup).Infof("deleted post exec flag entry from controller map for backup [%v/%v]", backup.Namespace, backup.Name)
```

Comment:
nit: good practice to print completion message after completion of the statement. Here we are printing deleted before calling delete.